src/libmscore/draw/painter.cpp

```diff
@@ -75,17 +74,34 @@ QPainter* Painter::qpainter() const
 
 bool Painter::end()
 {
-    return m_provider->end(m_name);
+    bool ok = m_provider->end(m_name);
     if (extended) {
         extended->end(m_name);
```

Comment:
Should we check the result here too?

Reply:
> Should we check the result here too?

There is no need. "Extended" is intended primarily for automated testing. In the near future, I will show what I have planned to do :)